8.0/Dockerfile

```diff
@@ -8,11 +8,11 @@
 
 # NOTE: This uses the November image due to an OpenSSL CVE. Using the quarterly image will resume in Q1
 ##### NOTE: PHP Versions < 8.1 need use Ubuntu 20.04 due OpenSSL/libssl. Please adjust accordingly #####
-FROM cimg/base:2022.11-20.04
+FROM cimg/base:2023.01
```

Comment:
this needs to be cimg/base:2023.01-20.04 since PHP versions < 8.1 are not compatible with OpenSSL 3, which is what 22.04 uses.

Updates to 8.0 should be sparse as it's only security updates now, though